codesets/mlflow/sklearn/conda.yaml

```diff
@@ -3,7 +3,6 @@ channels:
   - defaults
 dependencies:
   - python=3.6
-  - boto3
```

Comment:
This is because you've added `boto3` into the image, right?

Reply:
exactly, this was added by us. The original example does not really have this dependency.